content/S2016/square-event.md

```diff
@@ -0,0 +1,13 @@
+title: Square Tech Workshop: Becoming Ok with Okhttp
+Date: 2016-05-25 
```

Comment:
This is the date of the website post, not the date of the event. We should make sure to include the event date somewhere in here and update this to reflect today's date.